helm/docs/configuring-delegated-operators.md

```diff
@@ -36,44 +36,48 @@ has been deprecated in favor of `delegatedOperatorDeployments`.
 `delegatedOperatorExecutorSettings` has been marked for deletion
 for versions released after May 31st, 2025.
 
-This additional value allows users to deploy multiple instances
-of delegated operators, targeting different hardware or different
-use-cases.
+`delegatedOperatorDeployments` enables you to deploy multiple instances
+of delegated operators targeting different hardware or use-cases.
 
 ## Using `delegatedOperatorDeployments`
 
-`delegatedOperatorDeployments` allows you to apply a basic
-template to one or multiple delegated operator deployments.
-Values in `delegatedOperatorDeployments.template` will be applied to
-each deployment instance under `delegatedOperatorDeployments.deployments`.
+The values in `delegatedOperatorDeployments.template` will be applied to
+every deployment instance under `delegatedOperatorDeployments.deployments`.
+Per deployment values set in
+`delegatedOperatorDeployments.deployments.<DEPLOYMENT>`
+will override the template.
 
-For settings applied in both places, the following rules apply:
+For overlapping settings, the following rules apply:
 
-1. If the setting is a `map` object, the
-   `delegatedOperatorDeployments.template.SETTING` will be merged, key-wise,
-   with `delegatedOperatorDeployments.deployments.DEPLOYMENT.SETTING`.
-   For settings that are multiply defined, the
-   `delegatedOperatorDeployments.deployments.DEPLOYMENT.SETTING` will
+1. When the setting is a `map` object, the
+   `delegatedOperatorDeployments.template.<SETTING>` will be key-wise merged
+   with `delegatedOperatorDeployments.deployments.<DEPLOYMENT>.<SETTING>`.
+   For duplicate setting definitions, the
+   `delegatedOperatorDeployments.deployments.<DEPLOYMENT>.<SETTING>` will
    take precedence.
 
-1. If the setting is a `list` object, the
-   `delegatedOperatorDeployments.template.SETTING` will be ignored and the
-   `delegatedOperatorDeployments.deployments.DEPLOYMENT.SETTING` will
+1. When the setting is a `list` object, the
+   `delegatedOperatorDeployments.template.<SETTING>` will be ignored and the
+   `delegatedOperatorDeployments.deployments.<DEPLOYMENT>.<SETTING>` will
    take precedence.
 
-See [examples](#examples) for more information.
+See
+[examples](#examples)
+for more information.
 
 To enable delegated operators, add an object to `delegatedOperatorDeployments.deployments`:
 
 ```yaml
 delegatedOperatorDeployments:
+  enabled: true
```

Comment:
```suggestion
```

This isn't a thing; if there are no deployments configured, no delegated operators are deployed, and there are no delegated operator deployments configured by default

https://github.com/voxel51/fiftyone-teams-app-deploy/blob/main/helm/fiftyone-teams-app/templates/delegated-operator-instance-deployment.yaml#L3